ios/Classes/CountlyFlutterPlugin.m

```diff
@@ -359,7 +359,34 @@ - (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
         result(@"setLocation!");
         });
 
-    }else if ([@"enableCrashReporting" isEqualToString:call.method]) {
+    }else if ([@"setUserLocation" isEqualToString:call.method]) {
+         dispatch_async(dispatch_get_main_queue(), ^ {
+            NSDictionary* location = [command objectAtIndex:0];
+            NSString* gpsCoordinates =  location[@"gpsCoordinates"];
+            CLLocationCoordinate2D locationCoordinate = kCLLocationCoordinate2DInvalid;
+            if(gpsCoordinates && [gpsCoordinates containsString:@","]){
+                @try{
+                    NSArray *locationArray = [gpsCoordinates componentsSeparatedByString:@","];
+                    NSString* latitudeString = [locationArray objectAtIndex:0];
+                    NSString* longitudeString = [locationArray objectAtIndex:1];
+
+                    double latitudeDouble = [latitudeString doubleValue];
+                    double longitudeDouble = [longitudeString doubleValue];
```

Comment:
after this line. If any of the location values are 0, print warning of potential error and location string